src/main/java/nl/tudelft/contextproject/webinterface/ClientServlet.java

```diff
@@ -22,9 +22,7 @@
 	private static final long serialVersionUID = -8897739798010474802L;
 	
 	private static final String CONTENT_TYPE_JSON = "text/json";
-	
 	private static final Log LOG = Log.getLog("WebInterface");
-	
```

Comment:
Newline between static and nonstatic fields should be readded.

private static final Log LOG = ...
NEWLINE
private final transient WebServer server;